### DISCUSSION TITLE
Mac OS Sonoma Emulator not opening

### DISCUSSION THREAD

**Discussion by rnaufal32** [zepp-health/zeppos-forum#265] [Q&A]
Halo,

I'm having trouble displaying the simulator. The OS I'm using is MacOS Sonoma and the device I'm using is Macbook Air M1. I've attached a screenshot of the simulator below.

<img width="1680" alt="Screenshot 2023-11-09 at 11 04 19" src="https://github.com/zepp-health/zeppos-forum/assets/33546066/1177386a-30bb-48c2-830c-6f486da04cdd">

Please help, thank you.

**Comment by zeppos-hx** [comment]
The M-Series Mac require some additional operations, please follow this discussion and reinstall the virtual network driver, [https://github.com/orgs/zepp-health/discussions/262](https://github.com/orgs/zepp-health/discussions/262)
We will update this into the doc site soon

**Comment by MyoungXUE** [comment]
Hello, the documentation has been updated, please check the latest documentation.

https://docs.zepp.com/docs/guides/quick-start/environment/
<img width="956" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/800aaf71-1646-4fad-b263-be2f393ddfd4">

**Comment by rnaufal32** [comment]
I have tried the tutorial you provided, but it still doesn't show up.

I have disabled SIP and installed tun tap.

Here is the screenshot.

<img width="1680" alt="Screenshot 2023-11-10 at 07 18 08" src="https://github.com/zepp-health/zeppos-forum/assets/33546066/8bde215c-b3ce-4cfe-98ed-54b391627bb9">

**Reply by narky** [reply]
@rnaufal32 When you reboot you mac back, maybe you should click `uninstall tun and tap` and click `install` it again

**Reply by rnaufal32** [reply]
I have followed the tutorial you provided, but the simulator still has not appeared.

**Reply by MyoungXUE** [reply]
@rnaufal32 

Very strange problem, maybe we need to video link to solve this problem, please join our Slack community, private message us, communicate the time of the video conference.

Join Zepp Community Slack https://t.zepp.com/t/zepp-community-slack

**Reply by rnaufal32** [reply]
I realized that I hadn't allowed all of the Kernel Extensions. Now they're showing up. Thank you for your help.
